### PR TITLE
fixed withdraw issue on mayaplugin/playground

### PR DESCRIPTION
**Pull request by smartdev914**

### CODE REVIEW

File: playgrounds/vite/src/Liquidity/index.tsx [hunk 1]
```diff
@@ -79,12 +79,27 @@ export default function Liquidity({
...
+      tx = await skClient?.mayachain.withdraw({
+          assetValue: nativeAsset!,
+          percent: withdrawPercent,
+          from: "sym",
+          to : "cacao"
+      })
+      if(tx)
+        setWithdrawTx(tx);
+    }
+    else {
```
Comment (towanTG) [inline playgrounds/vite/src/Liquidity/index.tsx:93]: you dont need else here - just return early

File: playgrounds/vite/src/Liquidity/index.tsx [hunk 2]
```diff
@@ -79,12 +79,27 @@ export default function Liquidity({
   }, [nativeAssetValue, otherAssetValue, pluginMode]);
 
   const handleWithdraw = useCallback(async () => {
-    const ethAddress = await skClient?.thorchain.savings({
-      assetValue: nativeAsset!,
-      percent: withdrawPercent,
-      type: 'withdraw'
-    })
-    console.log("🚀 ~ handleWithdraw ~ ethAddress:", ethAddress)
+    let tx ;
```
Comment (towanTG) [inline playgrounds/vite/src/Liquidity/index.tsx:82]: would be better to have const inside the if and outside the if - dont use let, only if you really have to

**Review by chillios-dev — CHANGES_REQUESTED** [review]
Some integration issues with linter in few lines. Please recheck your environment.

File: playgrounds/vite/src/Liquidity/index.tsx (hunk 1 above)
Comment (smartdev914) [reply playgrounds/vite/src/Liquidity/index.tsx]: Fixed

File: playgrounds/vite/src/Liquidity/index.tsx (hunk 2 above)
Comment (smartdev914) [reply playgrounds/vite/src/Liquidity/index.tsx]: fixed

**Pull request merged**